base/special/exp.jl

```diff
@@ -211,7 +215,7 @@ end
     r = muladd(N_float, LogBo256U(base, T), x)
     r = muladd(N_float, LogBo256L(base, T), r)
     k = N >> 8
-    jU, jL = table_unpack(N&255 + 1)
+    jU, jL = table_unpack(N)
```

Comment:
why is this safe?

Reply:
Because the operation got moved into `table_unpack` - see further above:

```
ind = ind & 255 + 1 # 255 == length(J_TABLE) - 1
```